recipes/conda/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "4.1.5" %}
+
+package:
+  name: conda
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/conda/conda
+  git_tag: {{ version }}
+  git_depth: 1
```

Comment:
I'm inclined to agree. We just tend to not use git as a source. So I felt I should explain why we chose to do it here and provide options if it was deemed unacceptable.